python/sqlflow_submitter/tensorflow/explain_example.py

```diff
@@ -56,3 +58,37 @@
             is_pai=False,
             plot_type='bar',
             result_table="iris.explain_result")
+
+    # Train and explain DNNClassifier
+    train(datasource=datasource,
+          estimator=tf.estimator.DNNClassifier,
+          select="SELECT * FROM iris.train",
+          validate_select="SELECT * FROM iris.test",
+          feature_columns=feature_columns,
+          feature_column_names=feature_column_names,
+          feature_metas=feature_metas,
+          label_meta=label_meta,
+          model_params={
+              "n_classes": 3,
+              "hidden_units": [100, 100],
+          },
+          save="dnnmodel",
+          batch_size=100,
+          epochs=20,
+          verbose=0)
+
+    explain(datasource=datasource,
+            estimator_cls=tf.estimator.DNNClassifier,
+            select="SELECT * FROM iris.test LIMIT 10",
+            feature_columns=feature_columns,
+            feature_column_names=feature_column_names,
+            feature_metas=feature_metas,
+            label_meta=label_meta,
+            model_params={
+                "n_classes": 3,
+                "hidden_units": [100, 100],
+            },
+            save="dnnmodel",
+            is_pai=False,
+            plot_type='bar',
+            result_table="iris.explain_result")
```

Comment:
Will this overwrite the above test's result?

Reply:
No. They are saved to different files and DNN doesn't support explaining to table now.

Reply:
Maybe set `result_table=""` to avoid confusing?